code/modules/awaymissions/corpse.dm

```diff
@@ -364,6 +367,14 @@
 	outfit = /datum/outfit/job/clown
 	icon_state = "corpseclown"
 
+/obj/effect/mob_spawn/human/mime
```

Comment:
This isn't necessary - we don't support clown/mime content in ruins

Reply:
alrighty, I'll swap out the body for a botanist then, that's fitting enough.

Reply:
actually wait, does that encompass the ventriloquist dummy I already had in there considering that's part of the mime toolkit, or is that harmless?

Reply:
Naw that's fine